lighthouse-core/test/audits/byte-efficiency/duplicated-javascript-test.js

```diff
@@ -93,209 +93,114 @@ describe('DuplicatedJavascript computed artifact', () => {
             "subItems": Object {
               "items": Array [
                 Object {
-                  "sourceBytes": 48513,
+                  "sourceTransferBytes": 16009,
                   "url": "https://example.com/coursehero-bundle-1.js",
                 },
                 Object {
-                  "sourceBytes": 48513,
+                  "sourceTransferBytes": 16009,
                   "url": "https://example.com/coursehero-bundle-2.js",
                 },
               ],
               "type": "subitems",
             },
             "totalBytes": 0,
             "url": "",
-            "wastedBytes": 48513,
+            "wastedBytes": 16009,
           },
           Object {
             "source": "Control/assets/js/vendor/ng/select/angular-sanitize.js",
             "subItems": Object {
               "items": Array [
                 Object {
-                  "sourceBytes": 9135,
+                  "sourceTransferBytes": 3015,
                   "url": "https://example.com/coursehero-bundle-1.js",
                 },
                 Object {
-                  "sourceBytes": 9135,
+                  "sourceTransferBytes": 3015,
                   "url": "https://example.com/coursehero-bundle-2.js",
                 },
               ],
               "type": "subitems",
             },
             "totalBytes": 0,
             "url": "",
-            "wastedBytes": 9135,
+            "wastedBytes": 3015,
           },
           Object {
             "source": "node_modules/@babel/runtime",
             "subItems": Object {
               "items": Array [
                 Object {
-                  "sourceBytes": 5467,
+                  "sourceTransferBytes": 1804,
                   "url": "https://example.com/coursehero-bundle-1.js",
                 },
                 Object {
-                  "sourceBytes": 4410,
+                  "sourceTransferBytes": 1455,
                   "url": "https://example.com/coursehero-bundle-2.js",
                 },
               ],
               "type": "subitems",
             },
             "totalBytes": 0,
             "url": "",
-            "wastedBytes": 4410,
-          },
-          Object {
-            "source": "js/src/utils/service/amplitude-service.ts",
```

Comment:
feels weird that so many of these disappeared, are they really being compressed that well they fall below 200 bytes?

Reply:
The estimated transfer size for this source is 1022, which is still above the 200 sub-item threshold.

It's elided because the per-item threshold is 1024 :)

Reply:
Lowered the threshold for this test.

Reply:
oooooh the changed threshold wasn't for this test, gotcha gotcha, sorry for the noise :)